docs/concepts/storage/volumes.md

```diff
@@ -738,6 +739,53 @@ spec:
 
 For more information including Dynamic Provisioning and Persistent Volume Claims, please see the [StorageOS examples](https://github.com/kubernetes/kubernetes/tree/{{page.githubbranch}}/examples/volumes/storageos).
 
+### local
+This volume type is alpha in 1.7.
+
+A `local` volume represents a mounted local storage device such as a disk,
+partition or directory.
+
+Local volumes can only be used as a statically created PersistentVolume.
+
+Compared to HostPath volumes, local volumes can be used in a durable manner
+without manually scheduling pods to nodes, as the system is aware of the volume's
+node constraints.
+
+However, local volumes are still subject to the availability of the underlying
+node and are not suitable for all applications.
+
+The following is an example PersistentVolume spec using a `local` volume:
+``` yaml
```

Comment:
this doesnt render correctly because you need an empty line before the code block, and after the paragraph